.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: android
+sudo: true # travis-ci/travis-ci#5582 implies this could help against killed build-jobs
```

Comment:
That was also my intention. I just couldn't test it thoroughly, so time will tell if this is a solution. (An alternative would be trying to reduce memory consumption of gradle IF this is the problem.)